ebu_tt_live/documents/ebutt3.py

```diff
@@ -706,7 +706,7 @@ def create_compatible_document(self, *args, **kwargs):
             clock_mode=self._reference_clock.clock_mode,
             sequence_identifier=self._sequence_identifier,
             authors_group_identifier=self.authors_group_identifier,
-            sequence_number=self._last_sequence_number,
+            sequence_number=kwargs.get('sequence_number', self._last_sequence_number),
```

Comment:
Doesn't this _always_ use the specified sequence number if one was specified and the document is empty, even if there is a `_last_sequence_number`? I think that's the opposite of what we want, which is to use `_last_sequence_number` if it is `not None` or otherwise to try to use the specified `sequence_number` or if that fails, default to 1.

Reply:
To prefer  `_last_sequence_number` would mean that it is always used, as the constructor initializes it to `0`. However the variable is [updated by received documents, if lower than the present value](https://github.com/ebu/ebu-tt-live-toolkit/blob/release/2.1.2/ebu_tt_live/documents/ebutt3.py#L916). So in case e.g. a segment length of 2 seconds was configured and every 0.5s a document was received, this would result in output sequence numbers of 4, 8, 12 etc. 

As I'm not aware of other possible use cases of `create_compatible_document`, my intention was to not touch any other cases here and therefore to use the supplied sequence number here (if present), as already the case if the sequence has any segments stored.